thirdparty/lpeg/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ enable_language(C)
 assert_var_defined(CC)
 assert_var_defined(DYNLIB_CFLAGS)
 assert_var_defined(LUA_DIR)
-assert_var_defined(LUAJIT_LIB)
 
-set(BUILD_CMD_STR "${CC} ${DYNLIB_CFLAGS} -I${LUA_DIR}/src ${LUAJIT_LIB} -o lpeg.so lpcap.c lpcode.c lpprint.c lptree.c lpvm.c")
+# assuming lpeg.so is installed under $(OUTPUT_DIR)/rocks/lib/lua/5.1 and
+# libluajit.so is in $(OUTPUT_DIR)/libs
+set(BUILD_CMD_STR "${CC} ${DYNLIB_CFLAGS} -I${LUA_DIR}/src -Wl,-rpath,'../../../../libs' -lluajit -o lpeg.so lpcap.c lpcode.c lpprint.c lptree.c lpvm.c")
```

Comment:
In a perfect world, I'd avoid both relative paths and $ORIGIN in rpath and stick to absolute paths, but, clearly, in our case, that's not practical ;).

So, yeah, $ORIGIN is potentially slightly less random. Just have to make sure it doesn't get mangled by any buildscripts/tools and/or expanded by a shell somewhere along the line... ;).